models/markup.ts

```diff
@@ -64,7 +64,7 @@ let md = createMarkdownIt({ html: true, linkify: true })
       const actor = env.mentionedActors[handle];
       if (actor == null) return {};
       return {
-        class: "u-url mention",
+        class: "u-url mention bg-stone-300 rounded-lg p-1",
```

Comment:
This class is only applied to local posts, i.e., remote posts are not applied. We need to configure styles for `.prose a.mention:not(.hashtag)` in the *static/styles.css* file.